CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Documenter.jl changelog
 
+## Version `v0.26.0`
+
+* ![Feature][badge-feature] Documenter can now deploy from Gitlab CI to GitHub Pages with `Documenter.Gitlab`.
+
```

Comment:
Marked as 0.26.0 since I assume we're not going to add this to a patch release?

Reply:
Pre 1.0 you should use the "patch number position" for both patches and features (https://julialang.github.io/Pkg.jl/v1/compatibility/#compat-pre-1.0-1).

Reply:
Ah, good to know, thanks.